charts/nginx/templates/nginx-service.yaml

```diff
@@ -30,7 +30,11 @@ metadata:
     prometheus.io/scrape: "true"
     prometheus.io/port: {{ .Values.ports.metrics | quote }}
 spec:
-  type: LoadBalancer
+  type: {{ .Values.serviceType }}
```

Comment:
Looks good, but i was wondering how to check if user add by mistake wrong serviceType?

Reply:
Were you thinking of having a bunch of if statements and if the value was lb or something that we would put in LoadBalancer, and if they said nodeport we would put in NodePort, and ClusterIp?

I contemplated that but thought passing the value would look nicer and if a customer had an extension we wouldn't have to add explicit support for it.